website/docs/d/service_level_alert_helper.html.markdown

```diff
@@ -31,9 +31,9 @@ resource "newrelic_service_level" "foo" {
             from = "Transaction"
             where = "appName = 'Example application' AND (transactionType='Web')"
         }
-        good_events {
+        bad_events {
```

Comment:
Instead of replacing `bad_events` in the example,  can we include examples using  `good_events` and `bad_events`? The more useful examples we have, the better for our customers 🙂